src/main.cpp

```diff
@@ -59,6 +59,14 @@ int main(int argc, char *argv[])
 	// QML-GUI
 	//
 
+	if (strlen(qgetenv("QT_QUICK_CONTROLS_STYLE")) == 0)
+	{
+		qDebug() << "QT_QUICK_CONTROLS_STYLE not set, setting to Material";
+		qputenv("QT_QUICK_CONTROLS_STYLE", "Material");
+	}
+	qputenv("QT_QUICK_CONTROLS_1_STYLE", "Flat");
```

Comment:
Shouldn't we set it only if empty, too?

Reply:
I only use QQC 1 for the table which I would like to have in this design